db/migrate/20180926134519_fix_book_column_names.rb

```diff
@@ -0,0 +1,9 @@
+class FixBookColumnNames < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :books, :titulo, :title
```

Comment:
Thanks to change names, but your are not adding the requierment of the column here.

Reply:
Also is a very good practice to never modify a migration. But here you are in the same PR, same branch, so you can modify the previous migration file ;)
Tips: do a `bundle exec rake db:rollback` before modifying the migration ;)